src/trove/lists.scrbl

```diff
@@ -1962,6 +1829,30 @@ end
     end
     }
   ]
+  @function["distinct"
+    #:contract (a-arrow (L-of N) N)
+    #:args '(("lst" #f))
+    #:return N
```

Comment:
@jpolitz this can be extended beyond just `Number`, but it's not polymorphic because it expects equality to be implemented. What should we put in the documentation?